### PR TITLE
Created 1.7.4 release notes

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: _posts/release/2018-02-16-accumulo-1.7.4.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* [ACCUMULO-4619] - Add splits hung forever
+
+## Upgrading
+
+View the [Upgrading Accumulo documentation][upgrade] for guidance.
+
+## Testing
+
+[JIRA_174]: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12312121&version=12339577
+[upgrade]: https://accumulo.apache.org/docs/2.0/administration/upgrading
```
Comment (milleruntime) [inline _posts/release/2018-02-16-accumulo-1.7.4.md]: Did you intend for this to be linked to 2.0?  I am guessing this is the most up to date version of that page?

Comment (mikewalch) [reply _posts/release/2018-02-16-accumulo-1.7.4.md]: Yes, I intended to do this because it's the most up to date version.

**Comment by mikewalch** [comment]
> do you know how we determine which changes get put under Major changes?

For major changes, we typically write a few sentences about the change rather than a one-liner (as in the notable changes). I don't think there is any rule other than these are changes that we want users to know about.

File: _posts/release/2018-02-16-accumulo-1.7.4.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+meet the promises set forth to our users. Users of 1.6 or earlier that are seeking to upgrade to 1.7 should consider 1.7.4 as a starting point.
+
+## Major Changes
+
+## Other Notable Changes
+
+* [ACCUMULO-4817] - Update build plugins and parent POM
+* [ACCUMULO-4809] - Avoid blocking during session clean up
+* [ACCUMULO-4787] - Close input stream in AccumuloReplicaSystem
+* [ACCUMULO-4778] - Cache table name to id map
```
Comment (keith-turner) [inline _posts/release/2018-02-16-accumulo-1.7.4.md]: This issue was reverted for 1.7.

File: _posts/release/2018-02-16-accumulo-1.7.4.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+[JIRA_174]: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12312121&version=12339577
+[upgrade]: https://accumulo.apache.org/docs/2.0/administration/upgrading
+[user_manual]: https://accumulo.apache.org/1.7/accumulo_user_manual.html
+[javadoc]: https://accumulo.apache.org/1.7/apidocs/
+[examples]: https://accumulo.apache.org/1.7/examples/
+[semver]: http://semver.org/
+[guidelines]: https://github.com/apache/accumulo/blob/rel/1.7.3/README.md#api
+[patch_version]: http://semver.org/#spec-item-6
+[ACCUMULO-4458]: https://issues.apache.org/jira/browse/ACCUMULO-4458
+[FATE]: https://accumulo.apache.org/1.7/accumulo_user_manual.html#_fault_tolerant_executor_fate
```
Comment (keith-turner) [inline _posts/release/2018-02-16-accumulo-1.7.4.md]: This link is not used.

File: _posts/release/2018-02-16-accumulo-1.7.4.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Testing
+
+[JIRA_174]: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12312121&version=12339577
+[upgrade]: https://accumulo.apache.org/docs/2.0/administration/upgrading
+[user_manual]: https://accumulo.apache.org/1.7/accumulo_user_manual.html
+[javadoc]: https://accumulo.apache.org/1.7/apidocs/
+[examples]: https://accumulo.apache.org/1.7/examples/
+[semver]: http://semver.org/
+[guidelines]: https://github.com/apache/accumulo/blob/rel/1.7.3/README.md#api
```
Comment (keith-turner) [inline _posts/release/2018-02-16-accumulo-1.7.4.md]: Could update this for the 1.7.4 README which should eventually exist

File: _posts/release/2018-02-16-accumulo-1.7.4.md (hunk 2 above)
Comment (mikewalch) [reply _posts/release/2018-02-16-accumulo-1.7.4.md]: It's removed now. I was using the git history and didn't see the revert.  This pushed me to create a python script that generates release notes from the git history.  I pushed two more commits.  The first one contains the output generated by my script and the second the changes I made to create the release.

File: _posts/release/2018-02-16-accumulo-1.7.4.md (hunk 3 above)
Comment (mikewalch) [reply _posts/release/2018-02-16-accumulo-1.7.4.md]: It's removed now.

File: _posts/release/2018-02-16-accumulo-1.7.4.md (hunk 4 above)
Comment (mikewalch) [reply _posts/release/2018-02-16-accumulo-1.7.4.md]: Fixed in 9f9ad9e13d

**Comment by mikewalch** [comment]
Can I merge this? It's marked as draft so it won't show up on the site.

**Comment by keith-turner** [comment]
> Can I merge this? It's marked as draft so it won't show up on the site.

Since its marked draft, I think its good to merge it because it makes it easier for others to edit.

**Pull request merged**